backend/main.py

```diff
@@ -10,6 +12,50 @@
     env.reset(sys.argv[1])
     visualiser = Visualiser(env.realm)
 
+    #Temporary constants
+    car_size = 2
```

Comment:
Note another PR: we should calculate some realistic estimate for initial accelerations, truck sizes etc